.pre-commit-config.yaml

```diff
@@ -1,24 +1,24 @@
 repos:
-- repo: https://github.com/pre-commit/pre-commit-hooks
-  rev: v2.3.0
-  hooks:
-  - id: check-yaml
-    files: '\.(yml|yaml)$'
-  - id: end-of-file-fixer
-    files: '\.(yml|yaml)$'
-  - id: trailing-whitespace
-    files: '\.(yml|yaml)$'
-- repo: https://github.com/ansible/ansible-lint
-  rev: v6.13.1
-  hooks:
-  - id: ansible-lint
-    args: ["-c", ".ansible-lint.yml"]
-    files: '\.(yml|yaml)$'
-    language_version: python3
-    exclude: ansible/playbooks/prepare-data-dir.yml
-- repo: https://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.50.0
-  hooks:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+      - id: check-yaml
+        files: '\.(yml|yaml)$'
+      - id: end-of-file-fixer
+        files: '\.(yml|yaml)$'
+      - id: trailing-whitespace
+        files: '\.(yml|yaml)$'
+  - repo: https://github.com/ansible/ansible-lint
+    rev: v6.13.1
+    hooks:
+      - id: ansible-lint
+        args: [ "-c", ".ansible-lint.yml" ]
+        files: '\.(yml|yaml)$'
+        language_version: python3
+  #        exclude: ansible/prepare-data-dir.yml
```

Comment:
Should this dead code be deleted?

Reply:
I'm still holding out hope getting the ansible-lint pre-commit working 😞 and that commented line is necessary for it to not explode